<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportImage</class>
 <widget class="QDialog" name="DialogExportImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Image Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output Folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditFolderPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxOutputFormats">
     <property name="title">
      <string>Output Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxPDF">
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSVG">
        <property name="text">
         <string>SVG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPNG">
        <property name="text">
         <string>PNG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxJPG">
        <property name="text">
         <string>JPG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxTIFF">
        <property name="text">
         <string>TIFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxOutputPercentiles">
     <property name="title">
      <string>Output Percentile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxDisplayedPercentile">
        <property name="text">
         <string>Displayed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox95thPercentile">
        <property name="text">
         <string>95%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox99thPercentile">
        <property name="text">
         <string>99%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox99p9thPercentile">
        <property name="text">
         <string>99.9%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox99p99thPercentile">
        <property name="text">
         <string>99.99%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox100thPercentile">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxOutputGamma">
     <property name="title">
      <string>Output Gamma</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBoxGammaDisplayed">
        <property name="text">
         <string>Displayed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGamma0p5">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGamma1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGamma1p5">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Qualifiers</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBoxChannelNumberOffset">
        <property name="minimum">
         <number>-999</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Channel Number Offset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditFolderPath</tabstop>
  <tabstop>pushButtonBrowse</tabstop>
  <tabstop>checkBoxPDF</tabstop>
  <tabstop>checkBoxSVG</tabstop>
  <tabstop>checkBoxPNG</tabstop>
  <tabstop>checkBoxJPG</tabstop>
  <tabstop>checkBoxTIFF</tabstop>
  <tabstop>checkBoxDisplayedPercentile</tabstop>
  <tabstop>checkBox95thPercentile</tabstop>
  <tabstop>checkBox99thPercentile</tabstop>
  <tabstop>checkBox99p9thPercentile</tabstop>
  <tabstop>checkBox99p99thPercentile</tabstop>
  <tabstop>checkBox100thPercentile</tabstop>
  <tabstop>checkBoxGammaDisplayed</tabstop>
  <tabstop>checkBoxGamma0p5</tabstop>
  <tabstop>checkBoxGamma1</tabstop>
  <tabstop>checkBoxGamma1p5</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogExportImage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogExportImage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
